src/Workleap.OpenApi.MSBuild/ValidateOpenApiTask.cs

```diff
@@ -26,13 +30,14 @@ public sealed class ValidateOpenApiTask : CancelableAsyncTask
 
     protected override async Task<bool> ExecuteAsync(CancellationToken cancellationToken)
     {
+        var reportsPath = Path.Combine(this.OpenApiToolsDirectoryPath, "reports");
         var loggerWrapper = new LoggerWrapper(this.Log);
-        var processWrapper = new ProcessWrapper(this.OpenApiToolsDirectoryPath);
+        var processWrapper = new ProcessWrapper(this.StartupAssemblyPath);
```

Comment:
Do we want the startup path or the output? Let say they have a appsettings.json but they doesn"t copy on build, the tools will work but it won't be the same behavior as a real release.

Reply:
As far as I know .net looks for `appsettings.json` in build output folder by default. You can of course change this behavior but seems like an edge case. I would like to see an example of it.